### ISSUE TITLE
CORS Issue with processing job

### ISSUE THREAD

**Issue by NolanHill** [coyote-team/coyote-wp#87]
There seems to be a new issue that prevents the processing job from running. Not sure if this is related to a new WordPress version.
Got the following errors in my console when I clicked "Start processing job" in the coyote settings page:

- Access to fetch at 'https://processor.coyote.pics/jobs/' from origin 'https://www.cooperhewitt.org' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
- POST https://processor.coyote.pics/jobs/ net::ERR_FAILED      startProcessing @ settings.js?ver=5.7.1:116

**Comment by jkva** [comment]
When the instance was updated on which the processor runs, it was not started automatically. I've resolved this and will make it into an ops ticket. Thanks @NolanHill 👍
